client/app/bundles/course/assessment/submission/pages/LogsIndex/LogsHead.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import { TableBody, TableCell, TableRow } from '@mui/material';
+import { LogsMainInfo } from 'types/course/assessment/submission/logs';
+
+import { TableContainer } from 'lib/components/core/table';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import translations from './translations';
+
+interface Props {
+  with: LogsMainInfo;
+}
+
+const LogsHead: FC<Props> = (props) => {
+  const { t } = useTranslation();
+  const { with: info } = props;
+
+  return (
+    <TableContainer dense variant="outlined">
```

Comment:
Add a page header with `Access Logs` as the text and a back button.

Reply:
the back button will be used to go to the submissions page or the edit submission page? I'm guessing it's for going back to edit submission page but just to make sure

Reply:
Why edit submission page? The link to this page is found in submission index page no?

Reply:
I couldn't find any link towards the logs page from the submission index page

Reply:
Check `renderSubmissionLogsLink`. You should be able to access the page from the link there. And change the link in `renderSubmissionLogsLink` to react router's link.

Reply:
done